ui/src/routes/conversations/[id]/ConversationMessageInput.svelte

```diff
@@ -89,15 +102,17 @@
       </div>
     </label>
 
-    <div class="flex w-full flex-col">
+    <div class="flex w-full flex-col overflow-y-auto leading-6 max-h-[7.5rem]">
```

Comment:
```suggestion
    <div class="flex w-full flex-col overflow-y-auto leading-6">
```

This breaks rendering when you attach several files such that they overflow the max height. I think the max lines specified in the js is all we need.

https://github.com/user-attachments/assets/156daef8-43e4-4a46-a5a3-efbc5fd3f0a6